go/client/cmd_wot_vouch.go

```diff
@@ -0,0 +1,102 @@
+package client
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+	context "golang.org/x/net/context"
+)
+
+type cmdWotVouch struct {
+	assertion  string
+	message    string
+	confidence keybase1.Confidence
+	libkb.Contextified
+}
+
+func newCmdWotVouch(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	flags := []cli.Flag{
+		cli.StringFlag{
+			Name:  "m, message",
+			Usage: "A message about this user",
+		},
+		cli.StringFlag{
+			Name:  "vouched-by",
+			Usage: "Comma-separated list of keybase users who vouched for this user",
+		},
+		cli.StringFlag{
+			Name:  "verified-via",
+			Usage: "How you verified their identity: audio, video, email, other_chat, in_person",
+		},
+		cli.StringFlag{
+			Name:  "other",
+			Usage: "Other information about your confidence in knowing this user",
+		},
+		cli.IntFlag{
+			Name:  "known-for",
+			Usage: "Number of days you have known this user on keybase",
```

Comment:
there are other flags to come, right?

Reply:
yes, there's a ticket for the proof stuff.